extemporaneous_grading/translations

```diff
@@ -0,0 +1 @@
+locale/
```

Comment:
why using both translations and locale?

Reply:
For some reason if the two symbolic links are not present, the translations do not work.